src/Microsoft.Dnx.Tooling/Restore/ImplicitPackagesWalkProvider.cs

```diff
@@ -32,7 +32,15 @@ internal class ImplicitPackagesWalkProvider : IWalkProvider
             new RuntimeSpec("win10", "win81"),
             new RuntimeSpec("win10-x86", "win10", "win81-x86"),
             new RuntimeSpec("win10-x64", "win10", "win81-x64"),
-            new RuntimeSpec("win10-arm", "win10", "win81-arm"));
+            new RuntimeSpec("win10-arm", "win10", "win81-arm"),
+            new RuntimeSpec("unix", "any"),
```

Comment:
This is fallback logic, i.e. "linux" falls back to "unix" falls back to "any", or "darwin" falls back to "unix" falls back to "any"